src/main/webapp/js/script.js

```diff
@@ -342,43 +342,34 @@ function fillCourseInfoBox(courseInfo){
         var notes = courseInfo.notes;
 
         var termsOffered = "";
-        var fallIncluded = courseInfo.termsOffered.indexOf("f") >= 0;
-        var winterIncluded = courseInfo.termsOffered.indexOf("w") >= 0;
-        var summerIncluded = courseInfo.termsOffered.indexOf("s") >= 0;
-        if(courseInfo.termsOffered){
-            if(fallIncluded){
-                termsOffered = termsOffered + "fall";
-                if(winterIncluded)
-                    termsOffered += ", ";
-            }
-            if(winterIncluded){
-                termsOffered = termsOffered + "winter";
-                if(summerIncluded)
-                    termsOffered += ", ";
-            }
-            if(summerIncluded){
-                termsOffered = termsOffered + "summer";
-            }
-        }
+        // commenting this out as we are not supporting this feature for our MVP
```

Comment:
If we're not using this code, we night as well leave it out of master. It will still be saved in a commit.